### PR TITLE
GUVNOR-3200: [Guided Decision Table] Access control to prohibit business users from modifying columns

### PR DESCRIPTION
**Pull request by karreiro**
See https://issues.jboss.org/browse/GUVNOR-3200

---

permissoes

---

Part of an ensemble:
- https://github.com/AppFormer/uberfire/pull/861
- https://github.com/kiegroup/kie-wb-common/pull/1190
- https://github.com/kiegroup/drools-wb/pull/636
- https://github.com/kiegroup/kie-wb-distributions/pull/628
- https://github.com/kiegroup/kie-docs/pull/402

### CODE REVIEW

**Comment by karreiro** [comment]
@manstis I've created the doc https://github.com/kiegroup/kie-docs/pull/402
However, I'm still working in this PR. I'm getting an unexpected behavior when I put the default value as true.

**Comment by karreiro** [comment]
@manstis Changes applied here too :-)

File: kie-drools-wb-parent/kie-drools-wb-webapp/src/main/resources/security-module-admin.properties [hunk 1]
```diff
@@ -41,5 +41,6 @@ role.admin.permission.project.release=true
 role.admin.permission.repository.configure=true
 role.admin.permission.planner.available=true
 role.admin.permission.globalpreferences.edit=true
+role.admin.permission.guideddecisiontable.edit.columns=true
```
Comment (manstis) [inline kie-drools-wb-parent/kie-drools-wb-webapp/src/main/resources/security-module-admin.properties]: Not needed.. the default is ```true```

File: kie-wb-parent/kie-wb-webapp/src/main/resources/security-module-admin.properties [hunk 2]
```diff
@@ -41,5 +41,6 @@ role.admin.permission.project.release=true
 role.admin.permission.repository.configure=true
 role.admin.permission.planner.available=true
 role.admin.permission.globalpreferences.edit=true
+role.admin.permission.guideddecisiontable.edit.columns=true
```
Comment (manstis) [inline kie-wb-parent/kie-wb-webapp/src/main/resources/security-module-admin.properties]: Not needed.. the default is ```true```

**Comment by manstis** [comment]
@karreiro Please let me know when green and I'll merge them all.

**Comment by karreiro** [comment]
@manstis okay ;-)

**Comment by karreiro** [comment]
@manstis it’s green ;-)

**Pull request merged**